ehrapy/plot/_missingno_pl_api.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import annotations
+
+import missingno as msno
+import pandas as pd
+
+# Functionality provided by https://github.com/ResidentMario/missingno
+
+def missing_values_matrix(
+    adata: AnnData,
+    filter: str | None = None,
+    n: int = 0,
+    p: float = 0,
+    sort: str | None = None,
+    figsize: tuple = (25, 10),
+    width_ratios: tuple = (15, 1),
+    color: tuple = (0.25, 0.25, 0.25),
+    fontsize: float = 16,
+    labels: bool = True,
+    label_rotation: float = 45,
+    sparkline: bool = True,
+    categoricals: bool = False
+):  # pragma: no cover
+    """A matrix visualization of the nullity of the given AnnData object.
+
+    Args:
+        adata: :class:`~anndata.AnnData` object object containing all observations.
+        filter: The filter to apply to the matrix. Should be one of "top", "bottom", or None (default).
+        n: The max number of columns from the AnnData object to include.
+        p: The max percentage fill of the columns from the AnnData object.
+        sort: The row sort order to apply. Can be "ascending", "descending", or None.
+        figsize: The size of the figure to display.
+        width_ratios: The ratio of the width of the matrix to the width of the sparkline. Defaults to `(15, 1)`. Does nothing if `sparkline=False`.
+        color: The color of the filled columns. Default is `(0.25, 0.25, 0.25)`.
+        fontsize: The figure's font size. Default is 16pt.
+        labels: Whether or not to display the column names. Defaults to the underlying data labels when there are 50 columns or less, and no labels when there are more than 50 columns.
+        label_rotation: What angle to rotate the text labels to. Defaults to 45 degrees.
+        sparkline: Whether or not to display the sparkline. Defaults to True.
+        categoricals: Whether to include "ehrapycat" columns to the plot. Defaults to False.
+
+    Returns:
+        The plot axis.
+
+    Example:
+        .. code-block:: python
+
+            import ehrapy as ep
+
+            adata = ep.data.mimic_2(encoded=True)
+            ep.pl.missing_values_matrix(adata, filter='bottom', n=15, p=0.999)
+
+    Preview:
+        .. image:: /_static/docstring_previews/missingno_matrix.png
+    """
+
+    df = pd.DataFrame(adata.X, columns=adata.var_names)
+
+    if not categoricals:
+        filter_col = [col for col in df if not col.startswith('ehrapycat')]
```

Comment:
Could consider renaming `filter_cols` to something like `non_categorical_columns`. It's just a tiny bit more explicit. `filter_cols` or "filtered_cols" could by anything.